plugins/jobs/src/js/components/JobsBreadcrumbs.js

```diff
@@ -69,56 +49,67 @@ function getItemSchedule(item) {
   );
 }
 
-function getBreadcrumb(item, details = true) {
-  const id = item.getId();
-  const name = item.getName();
-  const link =
-    item instanceof Job ? `/jobs/detail/${id}` : `/jobs/overview/${id}`;
+function getBreadcrumb(item, id = "", name = "") {
+  console.log("getBreadcrumb", id);
+  const isDetailPage = item.id === id;
+  const link = isDetailPage ? `/jobs/detail/${id}` : `/jobs/overview/${id}`;
 
   return (
     <Breadcrumb key={id} title="Jobs">
       <BreadcrumbTextContent>
         <Link to={link}>{name === "" ? "Jobs" : name}</Link>
       </BreadcrumbTextContent>
-      {details ? getItemSchedule(item) : null}
-      {details ? getItemStatus(item) : null}
+      {/* {details ? getItemSchedule(item) : null */}
+      {isDetailPage ? <ItemStatus item={item} /> : null}}
     </Breadcrumb>
   );
 }
 
-function getBreadcrumbList(tree, item, details) {
-  if (item == null) {
+function getBreadcrumbList(item, details) {
+  if (item == null || item.id === "") {
     return [];
   }
 
-  return tree
-    .filterItems(function(currentItem) {
-      return (
-        item.getId() === currentItem.getId() &&
-        item.constructor === currentItem.constructor
-      );
-    })
-    .reduceItems(function(acc, currentItem) {
-      return acc.concat(getBreadcrumb(currentItem, details));
-    }, []);
+  return item.id
```

Comment:
It looks like we got rid of the tree here, but it does not look that we did that everywhere. In the case we are just dropping parts of the use of the tree, can we have a TODO ticket for the other places where we used the tree still and were not able to remove it.